src/components/todoFilter.js

```diff
@@ -0,0 +1,34 @@
+export const todoFliter = () => {
+    const $filters = document.querySelector('.filters');
+    $filters.addEventListener('click', filtering );
+}
+
+const filtering = ({target}) => {
+    const className = target.className;
+
+    const $todoList = document.getElementById('todo-list');
+    const $allTodo= $todoList.querySelectorAll('li');
+    const $completedTodo = $todoList.querySelectorAll('.completed')
+
+    if(className === 'all selected'){
+        display($allTodo);
```

Comment:
```suggestion
    if(className.contains('all'){
        display($allTodo);
```
이 부분은 이런 식으로 ```contains``` 메서드를 활용하는 방법도 좋은거 같습니다!